src/mumble/mumble.pro

```diff
@@ -392,7 +392,12 @@ win32 {
     LIBS *= -llibboost_system-mt -llibboost_thread-mt
   }
   win32-g++ {
-    LIBS *= -lboost_system-mt -lboost_thread_win32-mt
+    LIBS *= -lboost_system-mt
+    unix {
```

Comment:
A unix block inside win32-g++ really needs a comment to ensure people don't think it's dead code. I would think that.

Reply:
I agree.